### PR TITLE
Fix large logs

### PR DESCRIPTION
**Pull request by satr**

### CODE REVIEW

File: src/components/page-step/job-step-logs.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
+import * as PropTypes from 'prop-types';
+import AsyncResource from '../async-resource/simple-async-resource';
+import './style.css';
+import { usePollJobStepLogs } from './use-poll-job-step-logs';
+import { useGetJobStepFullLogs } from './use-get-job-step-full-logs';
+import { Log, LogDownloadOverrideType } from '../component/log';
+import { Accordion, Typography } from '@equinor/eds-core-react';
```
Comment (Pespiri) [inline src/components/page-step/job-step-logs.tsx:7]: Sort imports, keep external libraries separated by newline from internal includes, import styles.css last

```suggestion
import { Accordion, Typography } from '@equinor/eds-core-react';
import * as PropTypes from 'prop-types';

import { useGetJobStepFullLogs } from './use-get-job-step-full-logs';
import { usePollJobStepLogs } from './use-poll-job-step-logs';
import AsyncResource from '../async-resource/simple-async-resource';
import { Log, LogDownloadOverrideType } from '../component/log';

import './style.css';
```

File: src/components/page-step/job-step-logs.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+            <Accordion.Item isExpanded>
+              <Accordion.Header>
+                <Typography variant="h4">Log</Typography>
+              </Accordion.Header>
+              <Accordion.Panel>
+                <Log
+                  downloadOverride={downloadOverride}
+                  fileName={jobName + stepName + '.txt'}
+                  logContent={pollStepLogsState.data}
+                ></Log>
```
Comment (Pespiri) [inline src/components/page-step/job-step-logs.tsx:47]: Elements without children can be shortened

```suggestion
                <Log
                  downloadOverride={downloadOverride}
                  fileName={jobName + stepName + '.txt'}
                  logContent={pollStepLogsState.data}
                />
```

File: src/components/page-step/index.tsx [hunk 3]
```diff
@@ -221,32 +214,11 @@ export class PageStep extends Component<PageStepsProps, { now: Date }> {
...
-                  </Code>
-                ) : (
-                  <Typography>No logs</Typography>
-                )}
-              </AsyncResource>
+              <JobStepLogs
+                appName={appName}
+                jobName={jobName}
+                stepName={stepName}
+              ></JobStepLogs>
```
Comment (Pespiri) [inline src/components/page-step/index.tsx:221]: Elements without children can be shortened

```suggestion
              <JobStepLogs
                appName={appName}
                jobName={jobName}
                stepName={stepName}
              />
```

File: src/components/page-replica/index.tsx [hunk 4]
```diff
@@ -66,16 +79,19 @@ const PageReplica = ({
...
+              replica={replica}
+              downloadOverride={downloadOverride}
+              title={
+                <Typography>
+                  Replica <strong>{smallReplicaName(replicaName)}</strong>,
+                  component <strong>{componentName}</strong>
+                </Typography>
+              }
+            />
+          </>
```
Comment (Pespiri) [inline src/components/page-replica/index.tsx:94]: Remove unneeded extra fragment tag

```suggestion
          <Replica
            logState={pollLogsState}
            replica={replica}
            downloadOverride={downloadOverride}
            title={
              <Typography>
                Replica <strong>{smallReplicaName(replicaName)}</strong>,
                component <strong>{componentName}</strong>
              </Typography>
            }
          />
```

**Review by Pespiri — CHANGES_REQUESTED** [review]

File: src/components/page-step/index.tsx [hunk 5]
```diff
@@ -3,27 +3,22 @@ import * as PropTypes from 'prop-types';
...
   subscribeJob,
-  subscribeJobLogs,
   subscribePipelineRuns,
   unsubscribeJob,
-  unsubscribeJobLogs,
   unsubscribePipelineRuns,
 } from '../../state/subscriptions/action-creators';
 import { routeWithParams, smallJobName } from '../../utils/string';
-
 import './style.css';
```
[the remark this answers is not in the document]
Comment (satr) [reply src/components/page-step/index.tsx:16]: Fixed

File: src/components/page-step/index.tsx [hunk 6]
```diff
@@ -3,27 +3,22 @@ import * as PropTypes from 'prop-types';
...
 import { Dispatch } from 'redux';
```
[the remark this answers is not in the document]
Comment (satr) [reply src/components/page-step/index.tsx:17]: Fixed

File: src/components/page-step/index.tsx [hunk 7]
```diff
@@ -3,27 +3,22 @@ import * as PropTypes from 'prop-types';
 import { Component } from 'react';
 import { mapRouteParamsToProps } from '../../utils/routing';
```
[the remark this answers is not in the document]
Comment (satr) [reply src/components/page-step/index.tsx:4]: Fixed

File: src/components/page-replica/index.tsx [hunk 8]
```diff
@@ -1,17 +1,17 @@
-import { Typography } from '@equinor/eds-core-react';
 import * as PropTypes from 'prop-types';
 import { useEffect, useState } from 'react';
-
```
[the remark this answers is not in the document]
Comment (satr) [reply src/components/page-replica/index.tsx:4]: Fixed

File: src/components/page-scheduled-job/index.tsx [hunk 9]
```diff
@@ -19,6 +19,8 @@ import { getEnvsUrl, mapRouteParamsToProps } from '../../utils/routing';
 import { routeWithParams, smallScheduledJobName } from '../../utils/string';
 
 import './style.css';
+import { LogDownloadOverrideType } from '../component/log';
+import { useGetFullJobLogs } from './use-get-job-full-logs';
```
[the remark this answers is not in the document]
Comment (satr) [reply src/components/page-scheduled-job/index.tsx:23]: Fixed

File: src/components/page-scheduled-batch/index.tsx [hunk 10]
```diff
@@ -22,6 +22,8 @@ import { getEnvsUrl, mapRouteParamsToProps } from '../../utils/routing';
 import { routeWithParams, smallScheduledBatchName } from '../../utils/string';
 
 import './style.css';
+import { LogDownloadOverrideType } from '../component/log';
+import { useGetBatchFullLogs } from './use-get-batch-full-logs';
```
[the remark this answers is not in the document]
Comment (satr) [reply src/components/page-scheduled-batch/index.tsx:23]: Fixed

File: src/components/page-step/index.tsx [hunk 11]
```diff
@@ -3,27 +3,22 @@ import * as PropTypes from 'prop-types';
...
 import { connect } from 'react-redux';
-
```
[the remark this answers is not in the document]
Comment (satr) [reply src/components/page-step/index.tsx:6]: Fixed

**Pull request merged**